src/ResourceManagement/DataLake.Store/Microsoft.Azure.Management.DataLake.Store/Generated/Models/DataLakeStoreAccountUpdateParameters.cs

```diff
@@ -56,10 +60,11 @@ public partial class DataLakeStoreAccountUpdateParameters
         /// Possible values include: 'Consumption', 'Commitment_1TB',
         /// 'Commitment_10TB', 'Commitment_100TB', 'Commitment_500TB',
         /// 'Commitment_1PB', 'Commitment_5PB'</param>
-        public DataLakeStoreAccountUpdateParameters(IDictionary<string, string> tags = default(IDictionary<string, string>), FirewallState? firewallState = default(FirewallState?), TrustedIdProviderState? trustedIdProviderState = default(TrustedIdProviderState?), string defaultGroup = default(string), TierType? newTier = default(TierType?))
+        public DataLakeStoreAccountUpdateParameters(IDictionary<string, string> tags = default(IDictionary<string, string>), FirewallState? firewallState = default(FirewallState?), FirewallAllowAzureIpsState? firewallAllowAzureIps = default(FirewallAllowAzureIpsState?), TrustedIdProviderState? trustedIdProviderState = default(TrustedIdProviderState?), string defaultGroup = default(string), TierType? newTier = default(TierType?))
```

Comment:
Oh good catch, and this is interesting. I wanted to keep all firewall related stuff together, but I am open to suggestions here. Do you think we should always just append parameters to the back, even when we are going to preview? This breaking change won't come in to stable for some time, and it will result in a major version bump (from 1.* to 2.*). Thoughts?